exporters/otlp/otlpmetric/go.mod

```diff
@@ -0,0 +1,76 @@
+module go.opentelemetry.io/otel/exporters/otlp/otlpmetric
+
+go 1.14
+
+replace (
+	go.opentelemetry.io/otel => ../../..
+
+	go.opentelemetry.io/otel/sdk => ../../../sdk
+)
+
+replace go.opentelemetry.io/otel/exporters/otlp => ../
+
+replace go.opentelemetry.io/otel/metric => ../../../metric
+
+replace go.opentelemetry.io/otel/oteltest => ../../../oteltest
+
+replace go.opentelemetry.io/otel/trace => ../../../trace
+
+replace go.opentelemetry.io/otel/sdk/export/metric => ../../../sdk/export/metric
+
+replace go.opentelemetry.io/otel/sdk/metric => ../../../sdk/metric
```

Comment:
Can we move these below the `required` section to remain consistent? Maybe just remove these and re-run the `crosslink` util?